README.md

```diff
@@ -188,16 +182,16 @@ The following table lists the parameters for the `deployments` component and the
 | -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
 | `deployments.enabled` | Enable the component | `true` |
 | `deployments.automigrate` | Enable automatic database migrations at service start up | `true` |
-| `deployments.image.registry` | Docker image registry | `registry.mender.io` |
-| `deployments.image.repository` | Docker image repository | `mendersoftware/deployments-enterprise` |
-| `deployments.image.tag` | Docker image tag | `mender-2.6.0` |
+| `deployments.image.registry` | Docker image registry | `registry.mender.io` if `global.enterprise` is true, else `docker.io` |
+| `deployments.image.repository` | Docker image repository | `mendersoftware/deployments-enterprise` if `global.enterprise` is true, else `mendersoftware/deployments` |
+| `deployments.image.tag` | Docker image tag | `mender-2.7.0` |
 | `deployments.image.imagePullPolicy` | Docker image pull policy | `IfNotPresent` |
 | `deployments.replicas` | Number of replicas | `1` |
 | `deployments.affinity` | [Affinity map](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity) for the POD | `{}` |
-| `deployments.resources.limits.cpu` | Resources CPU limit | `500m` |
+| `deployments.resources.limits.cpu` | Resources CPU limit | `300m` |
 | `deployments.resources.limits.memory` | Resources memory limit | `128M` |
-| `deployments.resources.requests.cpu` | Resources CPU limit | `200m` |
-| `deployments.resources.requests.memory` | Resources memory limit | `50M` |
+| `deployments.resources.requests.cpu` | Resources CPU limit | `300m` |
+| `deployments.resources.requests.memory` | Resources memory limit | `64M` |
 | `deployments.service.name` | Name of the service | `mender-deployments` |
 | `deployments.service.annotations` | Annotations map for the service | `{}` |
 | `deployments.service.type` | Service type | `ClusterIP` |
```

Comment:
I think we should also expose (at least) `deployments.env.DEPLOYMENTS_PRESIGN_SECRET` so that configuration deployments doesn't break when replicas > 1.

Reply:
Nevermind, it's [already there](https://github.com/tranchitella/mender-helm/blob/1ac3d9f20597031dc218ead221b109f2e0fc1c1d/mender/templates/deployments-deploy.yaml#L82) :man_facepalming: